.github/workflows/build.yml

```diff
@@ -37,5 +42,34 @@ jobs:
     - name: Restore packages
       run: nuget restore UAssetGUI.sln
 
-    - name: Build
+    - name: Build All
       run: msbuild UAssetGUI.sln /p:Configuration=Release
+
+    - name: Build Publish Binary
+      run: dotnet publish --sc false -c Release -r win-x64 -p:PublishSingleFile=true -o ./build ./UAssetGUI/UAssetGUI.csproj

```

Comment:
Let me know if this is an unnecessary, that we could just use the executable from the Build All step instead.